lib/src/flutter_display_mode.dart

```diff
@@ -11,8 +11,15 @@ class FlutterDisplayMode {
   static const MethodChannel _channel = MethodChannel('flutter_display_mode');
 
   static Future<List<DisplayMode>> get supported async {
-    final List<dynamic> modes =
-        await _channel.invokeMethod<List<dynamic>>('getSupportedModes');
+    final List<dynamic>? modes = (await _channel.invokeMethod<List<dynamic>>(
+      'getSupportedModes',
+    )) ?? <dynamic>[];
+    if (modes == null) {
```

Comment:
`modes` will never be null. I think you will have to remove `?? <dynamic>[]`

Reply:
I was wondering the behavior that the API called from Android version below M (23), will it be a directly platform exception or a null result?

Reply:
It is market NotNull here https://github.com/ajinasokan/flutter_displaymode/blob/95889f9a18306f8e88cbdc00828596e4e7ea9934/android/src/main/java/com/ajinasokan/flutterdisplaymode/DisplayModePlugin.java#L120

Not sure how this will work in lower API levels though. I think we shouldn't add the PlatformException for these. I think we can keep it as it is since the bank list means there is no mode to switch.